sources/Inputs/src/NetworkManager.cpp

```diff
@@ -151,22 +151,21 @@ NetworkManager::buildTree() {
     }
 
     const auto& svarcs = networkVL->getStaticVarCompensators();
+    bool useDefaultModel = false;
```

Comment:
init of useDefaultModel shall be done at each loop

Reply:
Fixed. Added more elements to the PAR and DYD unit tests to verify.